src/Database/Mysql/Driver/Mysql.php

```diff
@@ -20,14 +20,18 @@
 
 class Mysql implements ProvidesDatabase
 {
-    public function configure(Tenant $tenant): array
+    public function configure(Tenant $tenant, $checkDirty = false): array
```

Comment:
Does the new argument conflict with the interface? I can't check on mobile.

Reply:
Don't know, it might but in that case, we can do 2 things:
- Adjust the contract
- Adjust the function so we check dirty on every configure attempt